src/core/useBVH.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { Mesh, Object3D } from 'three'
+import { acceleratedRaycast, computeBoundsTree, disposeBoundsTree } from 'three-mesh-bvh'
+
+export interface BVHOptions {
+  splitStrategy?: 'CENTER' | 'AVERAGE' | 'SAH'
+  verbose?: boolean
+  setBoundingBox?: boolean
+  maxDepth?: number
+  maxLeafTris?: number
+}
+
+export function useBVH(object3D: React.MutableRefObject<Mesh | undefined>, options?: BVHOptions) {
+  const ref = React.useRef<Object3D>()
```

Comment:
This ref seems a bit redundant? We're not using it for anything I don't think?

Reply:
Yeah, I took useHelper() as a template that was using the same logic. I'm ok with removing it